src/arkit/ARKitDevice.js

```diff
@@ -60,21 +60,21 @@ export default class ARKitDevice extends XRDevice {
 	}
 
 	static initStyles() {
-		window.addEventListener('DOMContentLoaded', () => {
-		  setTimeout(() => {
-			try {
-			  var styleEl = document.createElement('style');
-			  document.head.appendChild(styleEl);
-			  var styleSheet = styleEl.sheet;
-			  styleSheet.insertRule('.arkit-device-wrapper { z-index: -1; }', 0);
-			  styleSheet.insertRule('.arkit-device-wrapper, .xr-canvas { position: absolute; top: 0; left: 0; bottom: 0; right: 0; }', 0);
-			  styleSheet.insertRule('.arkit-device-wrapper, .arkit-device-wrapper canvas { width: 100%; height: 100%; padding: 0; margin: 0; -webkit-user-select: none; user-select: none; }', 0);
-			} catch(e) {
-			  console.error('page error', e);
-			}
-		  }, 1);
-		});    
-	  }
```

Comment:
What were `setTimeout` and `try-catch` for? I was not sure so removed. Let me know if I should haven't removed them.

Reply:
setTimeout was intended to delay the execution till after the event, just to decouple it.  I don't know why we did that, can't remember.  Probably not needed.